src/cmd/cmd.cpp

```diff
@@ -339,7 +339,14 @@ int main(int argc, char **argv)
         qFatal("Could not initialize account!");
         return EXIT_FAILURE;
     }
-    // check if the webDAV path was added to the url and append if not.
+
+    if (options.target_url.contains("webdav")) {
+        qWarning("Webdav is detected, even though it is deprecated.");
```

Comment:
@mgallien As you requested.

Reply:
Can you improve the message to not imply that webdav is deprectaed (webdav as the webdav protocol) but that the proper URL to use should include dav instead of webdav (as shown in the web UI) ?